docs/SPATIAL_FILTERING.md

```diff
@@ -0,0 +1,109 @@
+Spatial Filtering
+-----------------
+
+Starting with Kart 0.11.0, Kart supports spatially-filtered repositories. This is a Kart repository with an extra attached geometry describing a region that the user is interested in - any features that intersect that geometry are said to match the spatial filter, and they are shown. Conversely any features that do not intersect with it do not
+match, and are hidden.
+
+Spatial filtering has two main benefits:
+
+- Saves time, bandwidth, and disk space when cloning a repo and creating a working copy, by only downloading matching features.
+- Saves time loading the working copy into an editor and saves time for the user to focus in on their area of interest.
+
+The spatial filter is applied in three different ways:
+
+1. Only features that match the spatial filter are cloned during a `kart clone`.
+2. Kart commands that output features only output the features that match the spatial filter.
+3. The working copy is only populated with features that match the spatial filter.
+
+All three of these ways have some additional complexity, which will be explained in the following three sections. If this seems like more detail than you will need, skip ahead to learn how to [set the spatial filter](#Setting-the-spatial-filter)
```

Comment:
Move "Usage" up here, and put implementation details afterwards?

Reply:
Done